opentelemetry-exporter-geneva/geneva-uploader/src/bench.rs

```diff
@@ -0,0 +1,46 @@
+#[cfg(test)]
+mod benchmarks {
+    use crate::payload_encoder::lz4_chunked_compression::lz4_chunked_compression;
+    use criterion::{BenchmarkId, Criterion, Throughput};
+    use rand::{rngs::StdRng, Rng, SeedableRng};
+    use std::hint::black_box;
+
+    fn setup_test_data(size: usize) -> Vec<u8> {
+        let mut data = vec![0u8; size];
+        let mut rng = StdRng::seed_from_u64(42);
+        rng.fill(&mut data[..]);
+        data
+    }
+
+    /*
+        - Criterion benchmarks (lz4_chunked_compression, lz4_flex backend):
+
+            | Input size  | Median time        | Throughput         |
+            |-------------|--------------------|--------------------|
+            | 1 byte      | ~533 ns            | ~1.6 MiB/s         |
+            | 1 KiB       | ~597 ns           | ~1.5 GiB/s        |
+            | 1 MiB       | ~42 us           | ~22.1 GiB/s         |
```

Comment:
It would be good to identify what size would better represent the actual scenario.